stack.yaml

```diff
@@ -1,6 +1,18 @@
-resolver: lts-8.8
+resolver: nightly-2020-01-06
 packages:
 - '.'
 extra-deps:
-- xdot-0.3.0.1
-- svgcairo-0.13.2.0
+  - git: https://github.com/def-/xdot.git
```

Comment:
why not 0.3.0.2?

Reply:
The latest hackage release is `0.3.0.1`, `xdot` needs to be updated to support 8.8 before this PR lands.
I had switched to the git version in the hope that it had fixes for the segfaults.